recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
We were discussing something like that. Except it was a bit simpler. We would just override the value set in ` CPU_COUNT`.

However, this might take some time to implement and get right. So, I was trying to provide you a way out in the [comment above]( https://github.com/conda-forge/staged-recipes/pull/531/files/8d97b7ba1b38350dee5af0f89571002f3418715b#r62076418 ). Honestly, I'm pretty eager to see this deployed. So if we can settle this issue the simplest way possible for now, that would be ideal.